stable_baselines/common/tf_util.py

```diff
@@ -335,3 +466,38 @@ def _getter(getter, name, *args, **kwargs):
         val = getter(name, *args, **kwargs)
         return val
     return _getter
+
+
+# ================================================================
+# Logging
+# ================================================================
+
+def total_episode_reward_logger(rew_acc, rewards, masks, writer, steps):
```

Comment:
Note for myself: we should fix this one, it is not working properly when n_envs > 0 for a while now...